### ISSUE TITLE
ESP8266 compatibility

### ISSUE THREAD

**Issue by LucianTothezan** [Jupsi/logi_z906_wifi#5]
Hello. is this project capable of running on ESP8266 or the hardware is not powerful enough? Thank You in advance for Your feedback. Best regards.

**Comment by Jupsi** [comment]
Hi, i can't answer this question since i never owned a ESP8266.

You need 2 dedicated Serial Ports (4 Pins total)
3 ADC Ports (3+1 Pins total)

CPU wise i cant say if an ESP8266 is capable of running this but i would assume it can. Nevertheless i cant help if any library isnt compatible with the 8266. Good Luck trying it out :)

**Comment by LucianTothezan** [comment]
Thank You for the prompt reply. You gave the answer i believe in terms of compatibility: the ESP8266 has only one ADC pin so if the project needs 3 , there is no way to use the 8266. Best regards.

**Comment by LucianTothezan** [comment]
If i may have another question... You are powering the esp32 module from pin8? is that enough for the board in terms of amps once the comms are down? i do not see how...

L.E. as an idea...if pin 14 is really not used it could be linked to a permanent +V from the woofer`s board(+5v  if available, ideally +3.3V) and power the esp board(the pins table would be for pin 14 : 14    X    32  (if later on the esp is taken out of the circuit, the pin 14 from woofer would go directly to pin 14 on the console, but as it is not connected to anything, it should not pose any issues)
and another question please: if the esp would be only linked in parallel with the Tx Rx pins from the console, if a signal would be issued from the ESP32 modeule, would the console respond to it?(e.g. if volume up from esp32, would the leds on the controller follow the command?)

**Comment by Jupsi** [comment]
Hey, no its still powered by USB, Pin 8 is for checking Power on the Logitech System.

And i cant say if parallel hooking is possible on Serial, but with my Setup the Console is showing the correct LED's when switching Volume or Changing Channel / Effect. But its not parallel hooked its more like a daisy chain.

**Comment by LucianTothezan** [comment]
Ah, so You have independent power for the board. 

Need to search for daisy chain meaning :)

Thank You for taking the time for replying. I am willing to try your project so I ordered some esp32s and the 2 dsub connectors. I have used the Lewis Smallwood version until now but I would very much like to have controls from both the console and HA for the Z906. 
Best regards.

L.E: there is no way it could work just by linking the Tx RX in parallel as the TX should go to an Rx and a RX to an Tx in order to send receive data...
